permissions/mod.ts

```diff
@@ -2,6 +2,7 @@
 /**
  * Helpers for interacting with Deno's permissions system.
  * @module
+ * @deprecated (will be removed after 1.0.0) Use the Deno permission API instead
```

Comment:
```suggestion
 * @deprecated (will be in 1.0.0) Use the Deno permission API instead
```
I think this is clearer. I'd argue this is how all deprecations should be done.

Reply:
agreed

Reply:
Sidenote: perhaps, the "will be removed after..." check in [_tools/check_deprecation.ts](https://github.com/denoland/deno_std/blob/b34c646c45bcc3d4f7e797b78439a3cfd4d0b1b3/_tools/check_deprecation.ts) should be removed too (in a separate PR).